openedx/core/djangoapps/user_statistics/tasks.py

```diff
@@ -0,0 +1,11 @@
+import requests
+import json
+from celery.task import task
+from student.models import UserProfile
+
+
+@task(ignore_result=True)
+def count_students():
+    students_amount = UserProfile.objects.count()
+    r = requests.post('http://requestb.in/ptejf9pt', data=json.dumps({'number_of_students': students_amount}))
+    return r
```

Comment:
Please rename user_statistics to global statistics,
add module docstring, 
fix order of imports,
add function docstring.
Do not use 1letter variables (at least two letters)
Move URL into settings